test/suite/test_hs12.py

```diff
@@ -36,10 +37,15 @@ def timestamp_str(t):
 class test_hs12(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=2MB,statistics=(all),eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
+    key_format_values = [
+        # ('column', dict(key_format='r')),
```

Comment:
Can you add a similar comment that is present in test_hs09.py to remember to uncomment them when it is supported?

Reply:
The test seems to be working now after adding (uncommenting) the Key format as 'r'.